velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -144,5 +183,121 @@ TEST_F(ContainerRowSerdeTest, nested) {
   testRoundTrip(nestedArray);
 }
 
+TEST_F(ContainerRowSerdeTest, compareNullsInArrayVector) {
+  auto data = makeNullableArrayVector<int64_t>({
+      {1, 2},
+      {1, 5},
+      {1, 3, 5},
+      {1, 2, 3, 4},
+      {1, 2, std::nullopt, 4},
+      {1, std::nullopt, 5},
+  });
+  auto positions = serializeWithPositions(data);
+  auto arrayVector = makeNullableArrayVector<int64_t>({
+      {1, 2},
+      {1, 3},
+      {1, 5},
+      {std::nullopt, 1},
+      {1, 2, std::nullopt, 4},
+      {1, 5},
+  });
+  DecodedVector decodedVector;
```

Comment:
You can combine these 2 lines into one:

```
DecodedVector decodedVector(*arrayVector);
```

Ditto other places.

Reply:
Fixed